Resources/views/Action/redirect_session_to_checkout.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends layout ?: 'AppBundle:Payum:layout.html.twig' %}
+
+{% block payum_javascripts %}
+    {{ parent() }}
+
+    <script src="https://js.stripe.com/v3/"></script>
+
+    <script type="text/javascript">
+        var stripe = Stripe('{{ publishable_key }}');
+        stripe.redirectToCheckout({
+              sessionId: '{{ session_id }}'
+        }).then(function (result) {
+              // If `redirectToCheckout` fails due to a browser or network
+              // error, display the localized error message to your customer
+              // using `result.error.message`.
```

Comment:
what should be done on success?

Reply:
It's weird -- stripes forces us to do redirect via JS only. It's not possible to redirect to external stripe payment page from php code, therefore I'm creating new `Session`  at php side and doing redirect over here via `stripe.redirectToCheckout`

So answering your question -- on success it will redirect to stripe payment page. This particular comment in `on failure` section is from Stripe documentation, so I've left it as is. Normally it will be unreachable and if Customer PC suddenly loose connectivity and library user will want to handle it somehow, then this is the place.